include/dxc/DxilContainer/RDAT_SubobjectTypes.inl

```diff
@@ -74,99 +82,104 @@ RDAT_ENUM_END()
 
 #define RECORD_TYPE StateObjectConfig_t
 RDAT_STRUCT(StateObjectConfig_t)
-RDAT_FLAGS(uint32_t, hlsl::DXIL::StateObjectFlags, Flags)
+  RDAT_FLAGS(uint32_t, hlsl::DXIL::StateObjectFlags, Flags)
 RDAT_STRUCT_END()
 #undef RECORD_TYPE
 
 #define RECORD_TYPE RootSignature_t
 RDAT_STRUCT(RootSignature_t)
-RDAT_BYTES(Data)
+  RDAT_BYTES(Data)
 RDAT_STRUCT_END()
 #undef RECORD_TYPE
 
 #define RECORD_TYPE SubobjectToExportsAssociation_t
 RDAT_STRUCT(SubobjectToExportsAssociation_t)
-RDAT_STRING(Subobject)
-RDAT_STRING_ARRAY_REF(Exports)
+  RDAT_STRING(Subobject)
```

Comment:
Why do we need those space only changes? Clang format?

Reply:
We don't really need them.  The formatting was messed up for `RDAT_LibraryTypes.inl` due to typo, but fixing that results in flattening the entire file, which was even harder to read.  I thought it would be better to clang-format to wrap lines, but to indent for readability, as a compromise.  I couldn't find any other way to get clang-format to allow me to indent these for readability.

Since I changed that file this way, I took a look at the other files, and noticed this one was also harder to read from clang-format flattening, so I did the same thing here.  It's not necessary, and I could revert, but I'd rather have it this way, if that's acceptable to others.

Reply:
It is OK to do the change for readability.
But it would be nice to do that in separate PR so it is easier to review.